app/assets/stylesheets/styles/components/crops/components/tags-input.scss

```diff
@@ -0,0 +1,400 @@
+
+  &.ti-input-lg {
+
+    .input-group-lg {
+
+      .keywords {
+        min-height: 46px;
+      }
+    }
+
+    .tags {
+
+      .input-group-lg {
+
+        .keywords {
+
+          .tags {
+            border-radius: 0;
+          }
+        }
+      }
+
+      .tag-item {
+
+        .input-group-lg {
+
+          .keywords {
+
+            .tags {
+
+              .tag-item {
+                height: 38px;
+                line-height: 37px;
+                font-size: 18px;
+                border-radius: 0;
+              }
+            }
+          }
+        }
+
+        .remove-button {
+
+          .input-group-lg {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+
+                  .remove-button {
+                    font-size: 20;
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .input {
+
+        .input-group-lg {
+
+          .keywords {
+
+            .tags {
+
+              .input {
+                height: 38px;
+                font-size: 18px;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    &.ti-input-sm {
+
+      .input-group-sm {
+
+        .keywords {
+          min-height: 30;
+        }
+      }
+
+      .tags {
+
+        .input-group-sm {
+
+          .keywords {
+
+            .tags {
+              border-radius: 0;
+            }
+          }
+        }
+
+        .tag-item {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+                  height: 22px;
+                  line-height: 21px;
+                  font-size: 12px;
+                  border-radius: 0;
+                }
+              }
+            }
+          }
+
+          .remove-button {
+
+            .input-group-sm {
+
+              .keywords {
+
+                .tags {
+
+                  .tag-item {
+
+                    .remove-item {
+
+                      font-size: 16px;
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        .input {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .input {
+                  height: 22px;
+                  font-size: 12px;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+.keywords {
+
+  .tags {
+
+    .tag-item {
+
+      &.selected {
+        background: -webkit-linear-gradient(top, #febbbb 0%, #fe9090 45%, #ff5c5c 100%);
+        background: linear-gradient(to bottom, #febbbb 0%, #fe9090 45%, #ff5c5c 100%);
+      }
+
+      .remove-button {
+        margin: 0 0 0 5px;
+        padding: 0;
+        border: none;
+        background: none;
+        cursor: pointer;
+        vertical-align: middle;
+        font: bold 16px Arial, sans-serif;
+        color: #585858;
+
+        &:active {
+          color: #ff0000;
```

Comment:
`_variables.scss` is the only file they're defined in, right? I had to define a variable for this one since I couldn't find it (find and replace) in that one.